msgcntr/messageforums-api/src/java/org/sakaiproject/api/app/messageforums/MessageForumsMessageManager.java

```diff
@@ -40,7 +40,17 @@
     public void saveMessage(Message message);
 
     public void saveMessage(Message message, boolean logEvent);
-    
+    /**
+     * 
+     * @param message message
+     * @param logEvent logEvent
+	 * @param toolId id of the forums tool
+	 * @param userId user id
+	 * @param contextId context id'
+	 *
```

Comment:
Just noticed the spacing looks off in here. Sorry should have mentioned this before. Please use whatever is used or most prevalent in the file, be it tabs or spaces or whatever strange mixture of the two...